scripts/markdown/readme/languages/german.md

```diff
@@ -62,12 +62,18 @@ docker run -d --name nocodb -p 8080:8080 nocodb/nocodb:latest
   docker run -d -p 8080:8080 --name nocodb -v /local/path:/usr/app/data/ nocodb/nocodb:latest
   ```
 
-### Verwenden von NPM
+### Verwenden von NPM 
 
 ```
 npm install create-nocodb-app
 ```
 
+### Verwenden von NPX 
+
+```
+npx install create-nocodb-app
```

Comment:
You would get `npm ERR! could not determine executable to run`. Besides, please remove that. See the previous comment.

Reply:
Have corrected this after the merge.